### ISSUE TITLE
Confusing warning for loss function

### ISSUE THREAD

**Issue by roywei** [awslabs/keras-apache-mxnet#139]
Running a simple example under examples/mnist_cnn.py will produce the following warning:
```
/Users/lawei/Documents/Workspace/roywei/keras/keras/losses.py:128: UserWarning: MXNet Backend: If you are using a custom loss function and use slice operator in custom loss function, please set the **_keras_shape** attribute of the loss tensor explicitly.

Without this workaround, you may encounter shape mismatch errors in the broadcast operations.

For more details - https://github.com/awslabs/keras-apache-mxnet/issues/120


  warnings.warn('MXNet Backend: If you are using a custom loss function and use slice operator in custom '
```

Although it does not use any custom loss nor slice.
```
model.compile(loss=keras.losses.categorical_crossentropy,
              optimizer=keras.optimizers.Adadelta(),
              metrics=['accuracy'])
```

**Comment by sandeep-krishnamurthy** [comment]
No, this warning should be shown only if custom loss is used. Will look into it.

**Comment by roywei** [comment]
@sandeep-krishnamurthy  update: tried to change:
`
loss=keras.losses.categorical_crossentropy
`
to 
`loss='categorical_crossentropy'
`
warning is gone.

**Comment by roywei** [comment]
Closing as fixed in #158
